fuel-core-interfaces/src/executor.rs

```diff
@@ -50,6 +51,14 @@ pub struct ExecutionResult {
     /// The list of skipped transactions with corresponding errors. Those transactions were
     /// not included in the block and didn't affect the state of the blockchain.
     pub skipped_transactions: Vec<(Transaction, Error)>,
+    /// The status of the transactions execution included into the block.
```

Comment:
[nit] I think this sentence might need to be adjusted along the lines of "The status of executed transactions included in the block"